client/components/additionalFeatures.jsx

```diff
@@ -32,6 +32,21 @@ const Item = styled.div`
   margin-right: 10px;
 `;
 
+const H2 = styled.h2`
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 48px;
+  letter-spacing: -1px;
+`;
+
+const H4 = styled.h4`
+  font-weight: 400;
+  font-size: 24px;
+  line-height: 28px;
+  letter-spacing: -.3px;
+  margin-bottom: 5px;
+`;
```

Comment:
Is there a specific reason why you chose to use h2 and h4 elements here instead of span or p tags?

Reply:
Yes, I wanted to inherit the base css styling of h2 and h4 tags. If my understanding of this isn't right, please let me know 😅.

Reply:
That makes complete sense. I'm unsure of the base css styles of H tags but something I will look into!